.github/workflows/build.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     name: Clang-Format Check
     steps:
       - uses: actions/checkout@v4
-      - uses: jidicula/clang-format-action@v4.13.0
+      - uses: jidicula/clang-format-action@v4.14.0
         with:
           clang-format-version: '18'
```

Comment:
It seems v4.14.0 also brings support to Clang-Format 19, so also this could be bumped up in a separate PR or commit.

Reply:
Sure, do you want to do that? @cristian64

Reply:
I can do it in a few days, but _more_ than happy if you just do it now.